clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/provider/agent/EurekaCachingAgent.groovy

```diff
@@ -92,7 +93,7 @@ class EurekaCachingAgent implements CachingAgent, HealthProvidingCachingAgent, C
   @Override
   CacheResult loadData(ProviderCache providerCache) {
     log.info("Describing items in ${agentType}")
-    EurekaApplications disco = eurekaApi.loadEurekaApplications()
+    EurekaApplications disco = retry.retry({ eurekaApi.loadEurekaApplications() }, 3, 100, false)
```

Comment:
This is more to handle hitting a bad node than eureka being completely offline (hence the short retry interval)